src/sbt-test/bom/with-modules-per-scala/build.sbt

```diff
@@ -0,0 +1,24 @@
+import sbt._
+import sbt.Keys._
+
+inThisBuild(
+  Def.settings(
+    crossScalaVersions := Seq("2.13.4", "2.12.11"),
+    scalaVersion := (ThisBuild / crossScalaVersions).value.head,
+    organization := "com.lightbend",
+    version := "4.3.2.1"
+  )
+)
+
+lazy val billOfMaterials = project
+  .enablePlugins(BillOfMaterialsPlugin)
+  .settings(
+    name := "framework-sample-bom",
+    // Just one BOM including all cross Scala versions
+    crossVersion := CrossVersion.disabled,
+    // Create BOM in the 2.13 run
+    crossScalaVersions := crossScalaVersions.value,
```

Comment:
The comment and the setting don't fit together. It will create the pom.xml twice AFAICS.

Reply:
:+1: Updated comment to reflect it's created per scala version